libs/blocks/global-navigation/base.css

```diff
@@ -222,6 +222,7 @@
   .feds-navLink[class *= '-gradient'] {
     display: flex;
     border-radius: 4px;
+    font-weight: 600;
```

Comment:
Is it **always bolded**, or just in the gray gradient?

Reply:
I was informed that all gradients should be bolded.

Reply:
@mokimo - for context, we shouldn't be currently using any colorful gradient on production currently, it's still in a test phase. So I think it's safe to apply the rule to all such elements 

And come to think of it, could we confirm with the stakeholder whether they still see a need for all the other gradient classes? Maybe we could remove them already if that test failed.